pkgs/by-name/wo/wooting-udev-rules/package.nix

```diff
@@ -27,7 +27,7 @@ stdenv.mkDerivation {
     homepage = "https://help.wooting.io/article/147-configuring-device-access-for-wootility-under-linux-udev-rules";
     description = "udev rules that give NixOS permission to communicate with Wooting keyboards";
     platforms = platforms.linux;
-    license = "unknown";
+    license = lib.licenses.publicDomain;
```

Comment:
Ought to have a comment explaining the reasoning, and also the commit message is now wrong.

Reply:
Update commit message and added comment

Reply:
"thin**k**" (and "these" in commit message)